meetings/agenda.md

```diff
@@ -5,4 +5,52 @@
 
 Please file pull requests to add, or discuss items to add, to the agenda.
 
+[Continuation of the May TSC meeting](https://github.com/laminas/technical-steering-committee/blob/main/meetings/minutes/2024-05-06-TSC-Minutes.md)
+
 ## Items to Discuss
+
+### Create Media Kit 
+
+- create new logos or improve the current ones , for Laminas and Mezzio
+- svg format, various sizes and color
+- create few wallpapers
+- stickers from stickermule
+- banners and badges: "powered by Laminas " , or "powered by Mezzio"
+- official banner/seal for  "Official Laminas Commercial Vendor"
+- any other ideas related to marketing materials 
+  
+Responsible: Dotkernel organisation 
+
+Good to have it by: July 2024 TSC meeting 
+
+Where: a Media Kit page on getlaminas.org website 
+
+### getlaminas.org website update 
+
+We need approval for :
+- https://github.com/laminas/getlaminas.org/issues/162
+- https://github.com/orgs/laminas/projects/23/views/1
+
+Dotkernel organisation want to be involved in this, with Frank Brückner @froschdesign as project lead.
+
+The purpose is to have it ready on a new staging location , by July TSC meeting, for final approval .
+
+### OSS status for packages
+
+  For each package, add a file OSSMETADATA , with possible values 
+  - osslifecycle=active
+  - osslifecycle=maintenance
+  - osslifecycle=archived
+and add the badge to readme file
+Example:  ![OSS Lifecycle](https://img.shields.io/osslifecycle/dotkernel/api)
```

Comment:
This would really benefit to also have the date when the status was last reviewed.

Some packages might have very few recent code changes by design (so looking dead-ish). But showing that it was _recently_ re-confirmed as "active", would clear up all possible confusion. On the other hand a package might be so dead and forgotten that we didn't even remember to update its status to "archived". So at least showing that the status was last review was 5+ years ago, would let us know that something is wrong.